__tests__/components/PublicCommentCard.test.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is part of the Digital Planning Register project.
+ *
+ * Digital Planning Register is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Digital Planning Register is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Digital Planning Register. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { generateComment } from "@mocks/dprNewApplicationFactory";
+import { PublicCommentCard } from "@/components/PublicCommentCard";
+import { collapseTopicsByCharLimit } from "@/components/PublicCommentCard/PublicCommentCard.utils";
+import { TopicAndComments } from "@/types/odp-types/schemas/postSubmissionApplication/data/Comment";
+
+describe("Render PublicCommentCard", () => {
```

Comment:
You're tests aren't covering the skeleton version or the different comment types.

We should also mock any modules `capitaliseWord`, `formatDateTimeToDprDate`, `COMMENT_PUBLIC_TOPIC_OPTIONS`, `collapseTopicsByCharLimit` to make sure no issues in those affect these tests.